<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSG_NetworkClass</class>
 <widget class="QMainWindow" name="TSG_NetworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TSG_Network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txt_server">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>241</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_startServer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>开启服务器</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_closeServer">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>关闭服务器</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_ip_msg_server">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send_msg_server">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_contain_msg_server">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ip，为空则全发</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>内容</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_ip_file_server">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ip，为空则全发</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_filepath_file_server">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>170</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>J:\数据测试区\布心-xxwjpg\1\2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send_file_server">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_isStart_server">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>检查服务器是否启动</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_getClients_server">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>获得客户列表</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_sendfolder_server">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发文件夹</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <widget class="QPushButton" name="btn_connect_client">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>连接到服务器</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_ip_client">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>172.16.30.231</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_port_client">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>连接ip</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>端口</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_sendmsg_client">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_sendMsg_client">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_setfolder_client">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>J:\DownloadTester</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_setRecvFolder_client">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>设置接收文件夹</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_isStart_client">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>是否启动</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_stop_client">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>停止启动</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="txt_client">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>191</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TSG_Network.qrc"/>
 </resources>
 <connections/>
</ui>
